### PR TITLE
Fix warnings in vocabs and shapes

### PR DESCRIPTION
**Pull request by ndjc**
Fix warnings from XTurtle - unused prefixes, things declared twice
Also fixes issue #345.

### CODE REVIEW

**Comment by ndjc** [comment]
This PR addresses minor issues found in the Turtle plugin to Eclipse, validating the XML for the shapes; most of the modified files had just one common issue - unused prefix definitions.

**Comment by ndjc** [comment]
@berezovskyi  - PR awaiting review.

File: specs/perfmon/performance-monitoring-vocab.ttl [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@prefix qudt:  <http://qudt.org/1.1/schema/qudt> .
+@prefix qudt:  <http://qudt.org/1.1/schema/qudt#> .
```
Comment (berezovskyi) [inline specs/perfmon/performance-monitoring-vocab.ttl:1]: Good catch!

Comment (ndjc) [reply specs/perfmon/performance-monitoring-vocab.ttl:1]: The Eclipse plugin XTurtle caught it - it checks that all the vocabulary terms used are actually defined, looking in a cache of common vocabs it keeps. Sounds good - unfortunately the implementation is full of bugs, so I'd not recommend it as a common plugin for everyone.

**Pull request merged**